test/e2e/test_item.py

```diff
@@ -750,6 +742,120 @@ def test_partial_update_change_system_id(test_client):
+
+    item_patch = {"system_id": str(ObjectId())}
+
+    response = test_client.patch(f"/v1/items/{item_id}", json=item_patch)
+
+    assert response.status_code == 422
+    assert response.json()["detail"] == "The specified system ID does not exist"
+
+
+def test_partial_update_property_values(test_client):
+    """Test updating property values"""
+    response = test_client.post("/v1/systems", json=SYSTEM_POST_A)
+    system_id = response.json()["id"]
+
+    response = test_client.post("/v1/catalogue-categories", json=CATALOGUE_CATEGORY_POST_A)
+    catalogue_category_id = response.json()["id"]
+
+    response = test_client.post("/v1/manufacturers", json=MANUFACTURER_POST)
+    manufacturer_id = response.json()["id"]
+
+    # pylint: disable=duplicate-code
+    catalogue_item_post = {
+        **CATALOGUE_ITEM_POST_A,
+        "catalogue_category_id": catalogue_category_id,
+        "manufacturer_id": manufacturer_id,
+    }
+    response = test_client.post("/v1/catalogue-items", json=catalogue_item_post)
+    catalogue_item_id = response.json()["id"]
+
+    # pylint: enable=duplicate-code
+    item_post = {**ITEM_POST, "catalogue_item_id": catalogue_item_id, "system_id": system_id}
+    response = test_client.post("/v1/items", json=item_post)
+
+    item_patch = {
+        "properties": [
+            {"name": "Property A", "value": 12, "unit": "mm"},
+            {"name": "Property B", "value": True, "unit": None},
+            {"name": "Property C", "value": "20x15x10", "unit": "cm"},
+        ],
+    }
+
+    response = test_client.patch(f"/v1/items/{response.json()['id']}", json=item_patch)
+
+    assert response.status_code == 200
+
+    item = response.json()
+
+    assert item == {**ITEM_POST_EXPECTED, **item_patch, "catalogue_item_id": catalogue_item_id, "system_id": system_id}
+
+
+def test_partial_update_not_all_supplied_properties(test_client):
+    """Test updating a property when they are not all supplied"""
+    response = test_client.post("/v1/systems", json=SYSTEM_POST_A)
+    system_id = response.json()["id"]
+
+    response = test_client.post("/v1/catalogue-categories", json=CATALOGUE_CATEGORY_POST_A)
+    catalogue_category_id = response.json()["id"]
+
+    response = test_client.post("/v1/manufacturers", json=MANUFACTURER_POST)
+    manufacturer_id = response.json()["id"]
+
+    # pylint: disable=duplicate-code
+    catalogue_item_post = {
+        **CATALOGUE_ITEM_POST_A,
+        "catalogue_category_id": catalogue_category_id,
+        "manufacturer_id": manufacturer_id,
+    }
+    response = test_client.post("/v1/catalogue-items", json=catalogue_item_post)
+    catalogue_item_id = response.json()["id"]
+
+    # pylint: enable=duplicate-code
+    item_post = {**ITEM_POST, "catalogue_item_id": catalogue_item_id, "system_id": system_id}
+    response = test_client.post("/v1/items", json=item_post)
+
+    item_patch = {
+        "properties": [{"name": "Property A", "value": 15, "unit": "mm"}],
+    }
```

Comment:
I think it would be better if this supplies `Property B` and `Property C` instead of `Property A`. The value of `Property A` at a catalogue item level is `20` and the test creates an item and overrides the value of this property at an item level to `20`. So if we don't supply `Property A` in the `PATCH` request, it should technically inherit the value from the catalogue item and become `21, which is exactly what we need to verify.

`Property A` and `Property B` are not overridden when the item is created so it is expected that they will stay the same after the `PATCH`.